CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ IF( NOT WIN32)
                 SET(ICU_ROOT "/usr/local/opt/icu4c")
             ELSE()
                 MESSAGE(STATUS "set icu root homebrew")
-                SET(ICU_ROOT "/opt/homebrew/icu4c")
+                SET(ICU_ROOT "/usr/local/opt/icu4c")
```

Comment:
So we do not need the if else at all?
I am still wondering, how it fits to the homebrew docs. Can you check the binary type of the icu lib in this dir? So it it is an universal binary?

Reply:
It looks like it's `x86_64`?

```
% file /usr/local/opt/icu4c/lib/libicudata.71.1.dylib 
/usr/local/opt/icu4c/lib/libicudata.71.1.dylib: Mach-O 64-bit dynamically linked shared library x86_64
```

```
% file /usr/local/opt/icu4c/bin/icuinfo 
/usr/local/opt/icu4c/bin/icuinfo: Mach-O 64-bit executable x86_64
```

Reply:
Then it is probably bad for an arm build.
Also, icu4c is keg only, so it may be that it is not in the path. Check out my code in the arm build CI for discovering the position of the lib.
Also, does the linker complain at the end? And what is the type of the plugin?